### ISSUE TITLE
Problem when Installing

### ISSUE THREAD

**Issue by LucasRoot** [SpiderOak/Encryptr#275]
When I use the command "npm install" I get this: 

> npm WARN mocha-phantomjs@3.6.0 requires a peer of phantomjs@1.9.1 - 1.9.7-15 but none was installed.

I've already installed this packages with npm: phantomjs, mocha-phantomjs, phantomjs-prebuilt and gulp-mocha-phantomjs, but nothing works!

**Comment by devgeeks** [comment]
Yeah, it actually probably needs `phantomjs@1.9.x` installed locally or globally. I came across some of these issues updating the SpiderOak app. I'll need to give Encryptr a similar once over for use with modern npm.
